ui/raidboss/data/04-sb/eureka/eureka_hydatos.js

```diff
@@ -556,46 +572,33 @@
           return {
             en: 'Dark',
             de: 'Dunkel',
+            fr: 'Sombre',
             cn: '黑',
           };
         }
         if (data.bracelets == 'dark') {
           return {
             en: 'Light',
             de: 'Hell',
+            fr: 'Lumière',
             cn: '白',
           };
         }
       },
     },
     {
       // Note: These use 00:329e: lines, without any proper "gains effect" lines.
-      id: 'BA AV Eidos Relative Virtue Astral',
-      regex: Regexes.gameLog({ line: 'Relative Virtue gains the effect of Astral Essence', capture: false }),
-      regexDe: Regexes.gameLog({ line: 'Relative Tugend gains the effect of Arm des Lichts', capture: false }),
-      regexCn: Regexes.gameLog({ line: '相对的美德 gains the effect of 光之腕', capture: false }),
+      id: 'BA AV Eidos Relative Virtue Colors',
+      regex: Regexes.gameLog({ line: 'Relative Virtue gains the effect of (Astral|Umbral) Essence', capture: false }),
```

Comment:
This looks like you've reverted the changes I made here to split astral and umbral.  Can you undo that?

Reply:
I think there has been an update in the meantime, because the deletion is not voluntary. I restored the modified or deleted lines based on the current file (hydatos).